doc/user-guide.md

```diff
@@ -569,9 +572,9 @@ spec:
     certificate: {}
 ```
 
-In this scenario the operator will generate `Certificate` resource with common name of `myapp.test.svc` that can be used for service to service communication.
+In this scenario the operator will generate `Certificate` resource with common name of `my-app.test.svc` that can be used for service to service communication.
```

Comment:
Change to "
In this scenario,  the operator generates the `Certificate` resource with a common name of `myapp.test.svc`...".
Change to "In this scenario the operator generates `Certificate` resource with the common name of `my-app.test.svc`...".